setting-up/workload-onboarding/container/index.md

```diff
@@ -1,63 +1,32 @@
 ---
-description: How to use the Bacalhau Docker image
+description: How to use the Bacalhau CLI in Docker
 ---
 
-# Bacalhau Docker Image
+# Bacalhau CLI from Docker
 
-This documentation explains how to use the Bacalhau Docker image to run tasks and manage them using the Bacalhau client.
+This documentation explains how to use Bacalhau CLI from docker.
 
 ## Prerequisites
 
-To get started, you need to install the Bacalhau client (see more information [here](../../../getting-started/installation.md)) and Docker.
+Install the [Bacalhau CLI in Docker](../../..//getting-started/installation#step-1.1-install-the-bacalhau-cli).
 
-## 1. Pull the Bacalhau Docker image
-
-The first step is to pull the Bacalhau Docker image from the [Github container registry](https://github.com/orgs/bacalhau-project/packages/container/package/bacalhau).
-
-```
-docker pull ghcr.io/bacalhau-project/bacalhau:latest
-```
-
-Expected output:
-
-```shell
-latest: Pulling from bacalhau-project/bacalhau
-d14ccdd25413: Pull complete
-621f190d05c8: Pull complete
-Digest: sha256:3cda5619984de9b56c738c50f94188684170f54f7e417f8dcbe74ff8ec8eb434
-Status: Downloaded newer image for ghcr.io/bacalhau-project/bacalhau:latest
-ghcr.io/bacalhau-project/bacalhau:latest
-```
-
-You can also pull a specific version of the image, e.g.:
-
-```bash
-docker pull ghcr.io/bacalhau-project/bacalhau:v0.3.16
```

Comment:
why are we pinning to such an old version?

Reply:
Exactly that is why these instructions were removed from the guide and only the latest version is retained.